core/themes/seven/css/style.css

```diff
@@ -1586,6 +1586,25 @@ body.maintenance-page #logo {
   margin: 0 auto;
 }
 
+/* Links for the simplified login pages. */
+.user-form-links ul {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+.user-form-links li {
+  display: inline;
```

Comment:
You make it flex and then still inline? Why both approaches?

Reply:
My CSS-foo is really not very good :sweat_smile: Though seems like the overall appearance is in question so we'll need to get it sorted out what is acceptable from a visual standpoint, then I'll address code feedback.